usr.bin/ssh/sshd-session.c

```diff
@@ -648,7 +650,6 @@ recv_rexec_state(int fd, struct sshbuf *conf, uint64_t *timing_secretp)
 		fatal_f("rexec version mismatch");
 	if ((r = sshbuf_get_string(m, &cp, &len)) != 0 || /* XXX _direct */
 	    (r = sshbuf_get_u64(m, timing_secretp)) != 0 ||
-	    (r = sshbuf_froms(m, &hostkeys)) != 0 ||
```

Comment:
I didn't consider this before when I suggested it, but this makes this function non-symmetric with the sshd-session->sshd-auth config passing...

Reply:
i can skip this change if you think it's not improving things

Reply:
I think it's fine, just noting it in case we want to add a comment elsewhere noting the divergence